committees/models.py

```diff
@@ -128,38 +148,47 @@ def filter_this_year(res_set):
             year_start = date.today().replace(month=1, day=1)
             return res_set.filter(date__gte=year_start)
 
-        if ids is not None:
-            members = list(Member.objects.filter(id__in=ids))
-        else:
-            if current_only:
-                members = list((self.members.filter(is_current=True) |
-                                self.chairpersons.filter(is_current=True) |
-                                self.replacements.filter(is_current=True)).distinct())
-            else:
-                # What we actually would like to have is only member, active **during** that period
-                members = list((self.members.all() |
-                                self.chairpersons.all() |
-                                self.replacements.all()).distinct())
-
-        d = Knesset.objects.current_knesset().start_date if from_date is None else from_date
+        d = Knesset.objects.current_knesset().start_date if from_date is None \
+            else from_date
         meetings_with_mks = self.meetings.filter(
             mks_attended__isnull=False).distinct()
         all_meet_count = meetings_with_mks.filter(
             date__gte=d).count()
-        year_meet_count = filter_this_year(meetings_with_mks).count() if include_this_year else None
+        year_meet_count = filter_this_year(
+            meetings_with_mks).count() if include_this_year else None
         for m in members:
             all_member_meetings = m.committee_meetings.filter(committee=self,
                                                               date__gte=d)
             m.meetings_percentage = count_percentage(all_member_meetings,
                                                      all_meet_count)
             if include_this_year:
                 year_member_meetings = filter_this_year(all_member_meetings)
-                m.meetings_percentage_year = count_percentage(year_member_meetings,
-                                                              year_meet_count)
+                m.meetings_percentage_year = count_percentage(
+                    year_member_meetings,
+                    year_meet_count)
 
         members.sort(key=lambda x: x.meetings_percentage, reverse=True)
         return members
 
+    def all_members(self, current_only=False, ids=None):
```

Comment:
I am ambivalent about the names as well. But more so, I think the problem is deeper - `all_members`  should be rewritten as to return a QuerySet instead of a List, and then the dependent methods can be easily rewritten as an order by. 

In terms of names, the problem with `get` is that in django terms it usually means that one object will be returned, and it's misleading. I was trying to imitate a naming scheme of a custom `manager`, I think it makes more sense in the context. Do you maybe have other naming ideas that reflect that?